starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java

```diff
@@ -67,11 +67,9 @@ public void deleteTest() {
     inventory.setProductType(PRODUCT_TYPE);
     inventory = this.mongoTemplate.save(inventory);
     String id = inventory.getId();
-    Inventory deletedInventory = inventoryDAO.delete(id).get();
-    //Delete should return the inventory thus being non-null
-    Assert.assertNotNull(deletedInventory);
+    inventoryDAO.delete(id);
     //Inventory should no longer be in the database meaning searching by id should return null
-    Assert.assertNull(mongoTemplate.findById(id, Inventory.class));
+    Assert.assertNull(this.mongoTemplate.findById(id, Inventory.class));
```

Comment:
As our delete method will now return something, we should capture it and verify something is in the Optional container.